packages/earth-map/src/pages/earth/component.tsx

```diff
@@ -52,27 +33,9 @@ class EarthPage extends React.Component<IEarth> {
           }}
         />
 
-        <div className="l-sidebar">
-          <Sidebar />
-
-          <SidebarPanel>
-            <LayersDropdown native state={state}>
-              {({ x, ...props }) => (
-                <animated.div
-                  className="c-layers ng-section-background -active"
-                  style={{
-                    transform: x.interpolate((x) => `translate3d(0,${x}px,0)`),
-                    ...props,
-                  }}
-                >
-                  <Layers />
-                </animated.div>
-              )}
-            </LayersDropdown>
-            <Places />
-            {!!selected && <IndexSidebar />}
-          </SidebarPanel>
-        </div>
+        <Sidebar>
+          <SidebarLayoutSearch selected={!!selected} />
```

Comment:
can we just have Sidebar here? and not use children in the sidebar component? don't see why we need this abstraction

Reply:
I wanted the <Sidebar /> to be solely responsible for it's layout and the toggle functionality, not for the content inside it. In this way we could put anything inside the sidebar on different pages if that's needed in the future.